sdk/mobile-center-distribute/src/androidTest/java/com/microsoft/azure/mobile/distribute/ReleaseDetailsTest.java

```diff
@@ -136,16 +148,63 @@ public void nullReleaseNotes() throws JSONException {
         assertEquals(14, releaseDetails.getVersion());
         assertEquals("2.1.5", releaseDetails.getShortVersion());
         assertNull(releaseDetails.getReleaseNotes());
+        assertEquals(19, releaseDetails.getMinApiLevel());
         assertEquals(Uri.parse("https://download.thinkbroadband.com/1GB.zip"), releaseDetails.getDownloadUrl());
     }
 
     @Test(expected = JSONException.class)
+    public void missingApiLevel() throws JSONException {
+        String json = "{" +
+                "id: 42," +
+                "version: '14'," +
+                "short_version: '2.1.5'," +
+                "release_notes: 'Fix a critical bug, this text was entered in Mobile Center portal.'," +
+                "download_url: 'http://download.thinkbroadband.com/1GB.zip'" +
+                "}";
+        ReleaseDetails.parse(json);
+    }
+
+    @Test
+    public void acceptApiLevelAsString() throws JSONException {
```

Comment:
I can see swagger that its type is string. Pretty sad that it is not integer. :cry: